### ISSUE TITLE
Update login flow

### ISSUE THREAD

**Issue by MelissaAutumn** [thunderbird/appointment#513]
Right now we sort of drop the ball when it comes to any error messages or localization. When the oauth flow for fxa starts, we get the url from the backend and then yeet the user to that url. We then handle the callback on the backend, and if there's any errors it simply returns a blank page with the error message. 

At minimum we should localize the errors, and redirect them to the frontend with those error messages. We do something similar for failed google oauth errors.

**Comment by thunderbecca** [comment]
@MelissaAutumn will you confirm that this issue aims to _contextualize_ errors by improving the positioning into context of the UI rather than trying to _localize_ the error messages to align to international language requirements? From context I suspect it's the former and there is just a typo, but I'd like to verify.

**Comment by MelissaAutumn** [comment]
Yeah this ticket aims to surface the error messages, as I believe (it may have changed since July...) we don't show any real useful message, or sometimes we don't show any message. 

Additionally we should ensure that these messages are localized. If we get a google oauth error directly from google, we may only get it in English instead of German for example.

**Comment by davinotdavid** [comment]
Since the opening of this issue we have removed the whole FXA flow, changed it to OIDC / Keycloak and the oauth errors from external connections (during FTUE, for example, when setting up calendars) are being caught and displayed in EN / DE accordingly in their respective step.

Closing this issue for now but please re-open if this is still an issue!
